_data/data-sponsor.yml

```
@@ -0,0 +1,236 @@
+sponsor-home:
+  sponsor-heading: 2022 歌手赛赞助商
+  sponsor-levels:
+    - level-name: platinum
+      level-heading: <br>钻石赞助 虚位以待<br><br>
+
+sponsors_yj:
```

```suggestion
sponsors-yj:
```
Every other list uses dash, rather than underscore. Let's use dash here as well.

---

Removed as per the comment below.